samples/node-token-server/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2018 Coveo
```

Comment:
You should use this copyright format instead.

`Copyright (c) 2005-2018 Coveo Solutions Inc.`